### PR TITLE
Report size in bytes, and fix total space reporting

### PR DESCRIPTION
**Pull request by CameronDiver**
Change-type: patch
Signed-off-by: Cameron Diver <cameron@resin.io>

### CODE REVIEW

File: lib/index.coffee [hunk 1]
```diff
@@ -66,9 +66,9 @@ class DockerGC
 		.then =>
 			@docker.run 'alpine',
 				[
-					'/bin/sh',
-					'-c',
-					"df -k / | tail -n +2 | awk '{ print $3,$4 \"\\t\" }'"
+					'/bin/sh'
```
Comment (zvin) [inline lib/index.coffee]: Just wondering, would it work if we started df directly without /bin/sh ?

Comment (zvin) [reply lib/index.coffee]: Nevermind, you wouldn't be able to pipe if you did so.

File: lib/index.coffee [hunk 2]
```diff
@@ -80,13 +75,19 @@ class DockerGC
 					.on('end', () -> resolve(logStr))
 					.on('error', reject)
 			.then (spaceStr) ->
-				parts = spaceStr.split(/\s+/)
-				used = parseInt(parts[0])
+				# First split the lines, as we're only interested in the second one
+				lines = spaceStr.split(/\r?\n/).filter((l) -> l isnt '')
```
Comment (zvin) [inline lib/index.coffee]: lines = spaceStr.trim().split(/\r?\n/)

Comment (CameronDiver) [reply lib/index.coffee]: :p yeah...

**Pull request merged**